README_for_direct_archiver.md

```diff
@@ -11,49 +11,74 @@ Follow the steps:
     - prod - https://archiver.ingest.archive.data.humancellatlas.org
     
 2. Get the Archiver API key (`archiver_api_key`).
-```
-aws --region us-east-1 secretsmanager get-secret-value --secret-id ingest/archiver/wrangler/secrets --query SecretString --output text | jq -jr .{env}_archiver_api_key
-```
-Replace `{env}` with `dev`, `staging` or `prod` in the command above.
+   ```
+   aws --region us-east-1 secretsmanager get-secret-value --secret-id ingest/archiver/wrangler/secrets --query SecretString --output text | jq -jr .{env}_archiver_api_key
```

Comment:
We could use the `DEPLOYMENT_STAGE` environment variable from the `config/environment_{dev,staging,prod}` file.

Reply:
@amnonkhen I think you are mixing this repo: `ingest-archiver` with `ingest-kube-deployment` repo. We don't have those files in this repo.

Reply:
I know that it is in a different repo.
I suggested to use consistent environment variable names, so that we could source the environment script and have the relevant environment variables set.